src/transactions/feed/TransactionFeed.tsx

```diff
@@ -33,7 +34,12 @@ function TransactionFeed() {
 
   const cachedTransactions = useSelector(transactionsSelector)
   const allPendingTransactions = useSelector(pendingStandbyTransactionsSelector)
-  const allConfirmedStandbyTransactions = useSelector(completedStandbyTransactionsSelector)
+  const allFailedTransactions = useSelector(failedStandbyTransactionsSelector)
+  const allCompletedStandbyTransactions = useSelector(completedStandbyTransactionsSelector)
+  const allConfirmedStandbyTransactions = [
```

Comment:
it seems like we want to treat successful and failed completed transactions the same right? so is there any need to create a separate failed tx's selector? (should we just add failed tx's to the `completedStandbyTransactionsSelector`?)